src/main/scala/org/scalameter/persistence/SerializationPersistor.scala

```diff
@@ -3,62 +3,28 @@ package persistence
 
 
 
-import java.util.Date
 import java.io._
-import collection._
 import Key.reports._
 
 
 
-case class SerializationPersistor(path: File) extends Persistor {
+case class SerializationPersistor(path: File) extends IOStreamPersistor[ObjectInputStream, ObjectOutputStream] {
```

Comment:
Could you please elaborate more on that - e.g. what kind of unit test do you expect?
I'm asking because there is already a unit test ([SerializationPersistorTest](https://github.com/kjanosz/scalameter/blob/json-serialization-persistor/src/test/scala/org/scalameter/persistence/SerializationPersistorTest.scala)) along others *PersistorTest's that check if given persistor correctly serializes and deserializes output.